### DISCUSSION TITLE
request help: jwt功能只做token验证，关闭获取token的接口

### DISCUSSION THREAD

**Discussion by wjmmjr** [apache/apisix#5180] [Q&A] [answered]
### Issue description

能不能开启jwt功能，关闭获取jwt token的接口，客户自己根据我们提供的key等信息自己生成token，我们只做token验证

### Environment

- apisix version (cmd: `apisix version`):
- OS (cmd: `uname -a`):
- OpenResty / Nginx version (cmd: `nginx -V` or `openresty -V`):
- etcd version, if have (cmd: run `curl http://127.0.0.1:9090/v1/server_info` to get the info from server-info API):
- apisix-dashboard version, if have:
- the plugin runner version, if the issue is about a plugin runner (cmd: depended on the kind of runner):
- luarocks version, if the issue is about installation (cmd: `luarocks --version`):

**Comment by laokan** [comment, marked as answer]
`curl -i -X PUT \
   -H "Content-Type:application/json" \
   -H "X-API-KEY:edd1c9f034335f136f87ad84b625c8f1" \
   -d \
'{
    "interceptors": [
        {
            "name": "ip-restriction",
            "conf": {
                "whitelist": ["10.0.0.0/8"]
            }
        }
    ]
}' \
 'http://apisix:9080/apisix/admin/plugin_metadata/prometheus'`

可以利用 interceptors 把url 保护一下
